front/package.json

```diff
@@ -10,7 +10,7 @@
     "@types/node": "^12.20.30",
     "@types/react": "^17.0.29",
     "@types/react-dom": "^17.0.9",
-    "antd": "^4.17.3",
+    "antd": "^4.22.3",
```

Comment:
I had to upgrade `antd` to correctly use [dropdown menus](https://ant.design/components/dropdown/). W/o upgrade, I confronted an odd type error which I think related to some minor updates since `4.17.3`.